src/token.def

```diff
@@ -134,7 +135,7 @@ WABT_TOKEN(Throw, "throw")
 WABT_TOKEN(Try, "try")
 WABT_TOKEN(Unary, "UNARY")
 WABT_TOKEN(Unreachable, "unreachable")
-WABT_TOKEN_FIRST(Opcode, AtomicLoad)
```

Comment:
Was AtomicLoad not supposed to be here at all?

Reply:
Mmm, I think this was a bad change that snuck in.  I will make sure it's not there in the revised patch and regenerate the lexer.